scripts/cloudbuild_k8s_generator.py

```diff
@@ -101,9 +78,12 @@
   # https://cloud.google.com/cloud-build/docs/overview#build_configuration_and_build_steps
   - 'EXTRA_DOCKER_PARAMS=--net cloudbuild'
   dir: k8s/{{ solution }}
+  entrypoint: bash
   args:
-  - make
-  - crd/install
+  - -exc
+  - |
+    gcloud container clusters get-credentials '${_CLUSTER_NAME}' --zone '${_CLUSTER_LOCATION}' --project '$PROJECT_ID'
```

Comment:
isn't this already done in the `Initialize Credentials` step?

Reply:
I am also intrigued. I went over a battery of trial and error attempts. The gcloud is unable to authenticate in this step if the credentials are not collected.

Another interesting fact, the credentials tend to propagate as long as we use the same image in subsequent steps. If they run concurrently, the order might mess it up as well.

I am actually revisiting this whole setup due to other problems. 
- Cloud build limits us to 30 concurrent builds, which makes it impossible to have them all pass.
- Each build triggers yet another build, which aggravates the quota limit. We probably have to unnest them for cross solution changes to pass.